### ISSUE TITLE
sparql endpoint chunked list component does not persists "columns as literals" 

### ISSUE THREAD

**Issue by nvdk** [linkedpipes/etl#804]
The sparql endpoint chunked list component has a configuration option to treat certain input columns as literals, as seen here:
![Screenshot from 2020-05-27 14-32-19](https://user-images.githubusercontent.com/878686/83019313-036f0e80-a027-11ea-9cf1-44215d4c8397.png)

This currently doesn't seem to work as expected:
 * filling in the column header in this field results in it being blank when reopening the configuration. 
 * the constructed query does not use the column as a literal, but rather as a uri

**Comment by skodapetr** [comment]
@nvdk Could you please share "non"-working example - so we can faster reproduce and fix the issue? 

Your description seems to be sufficient, but should we have your example we can be sure that we really resolved your issue.

**Comment by jakubklimek** [comment]
@skodapetr Actually, it can be seen on [this](https://demo.etl.linkedpipes.com/#/pipelines/edit/canvas?pipeline=https:%2F%2Fdemo.etl.linkedpipes.com%2Fresources%2Fpipelines%2Fcreated-1486384033928&execution=https:%2F%2Fdemo.etl.linkedpipes.com%2Fresources%2Fexecutions%2F1590571321698-891-cb18db11-4e7e-4720-a963-b560a9f0a664) demo execution - in the second component, you can fill the "columns as literals" field, but on reopening the window, it is empty.

**Comment by skodapetr** [comment]
@nvdk @jakubklimek They are chips you need to press enter after typing them in. Otherwise, the values are not saved.

So perhaps the solution might be to improve the documentation?

**Comment by jakubklimek** [comment]
@skodapetr Right, I completely forgot about that. Yes, this should be hinted in the dialog.
